ouroboros-consensus/docs/interface-CHANGELOG.md

```diff
@@ -0,0 +1,117 @@
+mistake in an old entry, we address it by adding new lines at the top of that
+log entry. The new list item within the old entry should follow this template `-
+Errata Circa 2022-12-30: The original discussion, still found below,
+oversimplifies the ...`. Such errata only correct mistakes in the original log
+entry; crucially, the the code relevant to that entry is not changing. Such code
+changes would justify their own new log entry instead of adding errata to an old
+one. Because of this scheme, the errata dates, unlike the top-level entry dates,
+may appear out of chronological order.
+
+The internals of each entry are organized similar to
+https://keepachangelog.com/en/1.1.0/, adapted to our plan explained above.
+
+## Circa 2021-07-29
+
+### Added
+
+- `ouroboros-consensus/docs/interface-CHANGELOG.md`. Now that this file exists,
+  entries added to it by later PRs will be narrower than this entry. This entry
+  should be the only one that is bootstrapping the file by discussing multiple
+  recent changes.
+
+- The `*MaxTxCapacityOverrides` fields to the `ProtocolParams*` record types,
+  one per ledger era (eg `byronMaxTxCapacityOverrides`,
+  `allegraMaxTxCapacityOverrides`, etc). In particular, these record types are
+  used to organized the lost list of arguments to
+  `Ouroboros.Consensus.Cardano.Node.protocolInfoCardano`. Downstream libraries
+  invoke that function.
+
+  Integration advice: These fields have type
+  `Ouroboros.Consensus.Mempool.TxLimits.Overrides` and therefore can be
+  defaulted with `mempty` or more explicitly with `mkOverrides
+  noOverridesMeasure` (exported by that same module).
+
+  Eventually, the SPOs should be able to set options in configuration files that
+  influence the value of these new override fields in order to restrict the
+  various notions of total transaction size allowed in a block they forge. These
+  limits are already provided by the ledger state, but the SPO may have reason
+  to reduce that limit via this override. The fields are per-era because 1) some
+  eras introduce new dimensions/components of the "transaction size" concept and
+  2) even for a single such component eg script complexity, it can be reasonable
+  for the SPO to set different limits for different eras.
+
+### Changed
+
+- On-the-wire content of Alonzo transactions. Previously they were sent across
+  the wire without the flag indicating whether or not their carried scripts were
+  expected to pass. Now that flag is included on-the-wire.
+
+  The original motivation is so that the local node can reject mistakenly
+  invalid scripts from the local client when the client sent the positive flag
+  but the scripts failed. This is a courtesy to our local client, catching
+  mistakes that otherwise would have incurred on-chain penalties.
+
+  Beyond the original motivation, it is also useful to receive this flag
+  untrusted peers. If they send the wrong flag, we correct it before propagating
```

Comment:
I feel like part of this sentence was deleted. Do you mean "... useful to receive this flag in order to detect untrusted peers"?

Reply:
... or a missing "from"

Reply:
Fixed, thanks!